tests/integration/ceph_base_mgr_test.go

```diff
@@ -0,0 +1,36 @@
+/*
+Copyright 2016 The Rook Authors. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package integration
+
+import (
+	cephclient "github.com/rook/rook/pkg/daemon/ceph/client"
+	"github.com/rook/rook/tests/framework/utils"
+)
+
+func devicels(k8sh *utils.K8sHelper, namespace string) (error, string) {
+	cmd := cephclient.NewCephCommand(k8sh.MakeContext(), namespace, []string{"orchestrator", "device", "ls"})
```

Comment:
```suggestion
	cmd := cephclient.NewCephCommand(k8sh.MakeContext(), namespace, []string{"orch", "device", "ls"})
```
(for octopus)

Reply:
Thx!! I'm aware. Other thing to decide. Do we want different tests for different Ceph versions , or we should use the same tests changing the command depending of the Ceph version?